### PR TITLE
added vendor Black Crow AI analytics (#31449)

### PR DESCRIPTION
**Pull request by raymondborkowski**
Add Black Crow AI Analytics integration for our partners to easily integrate.

@ampproject/wg-components

Link to issue: https://github.com/ampproject/amphtml/issues/31449

### CODE REVIEW

File: extensions/amp-analytics/0.1/vendors/blackcrowai.json
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors/blackcrowai.json:12]: default value is `true`. only need to specify `useBody` here.

File: extensions/amp-analytics/0.1/vendors/blackcrowai.json
Comment (zhouyx) [inline extensions/amp-analytics/0.1/vendors/blackcrowai.json:21]: is page_id customizable? can this be included in the request instead? Your call

File: extensions/amp-analytics/0.1/vendors/blackcrowai.json
Comment (raymondborkowski) [reply extensions/amp-analytics/0.1/vendors/blackcrowai.json]: Great, updated, thanks! 👍

File: extensions/amp-analytics/0.1/vendors/blackcrowai.json
Comment (raymondborkowski) [reply extensions/amp-analytics/0.1/vendors/blackcrowai.json]: No, it is a static value important for us to understand this traffic is coming from AMP pages. I did update the value though, thanks! 👍

**Review by zhouyx — APPROVED** [review]

**Comment by raymondborkowski** [comment]
@zhouyx a unit test seems to be failing in Travis. 

It looks like ` http://help.pressboard.ca/publisher-resources/getting-started/implementing-google-amp` is a 404'ing. This is unrelated to my changes but causing the test to fail ` gulp check-links --local_changes`. Will this hold up my merge or should I do something about it?

**Comment by zhouyx** [comment]
Ah! You're right. You can try to wrap all unresolved url with backticks. For example
```
### mediarithmics

Type attribute value: `mediarithmics`

Adds support for mediarithmics. More information and configuration details can be found at `https://developer.mediarithmics.com`.
```

I can also help to fix that, but that would require an owner to approve the PR. That means we will need to wait until next week.

**Pull request merged**
